### ISSUE TITLE
Invalid Expression table.ROWTIME when table was created with backticks around name

### ISSUE THREAD

**Issue by tkaven** [confluentinc/ksql#2176]
```
ksql> CREATE TABLE `ks_hgame_projects_n111_table` (userid BIGINT, iscancel INTEGER, tprice DOUBLE, bonus DOUBLE, btimestamp BIGINT, wtimestamp BIGINT, dtimestamp BIGINT, pid BIGINT, lid INTEGER, mid BIGINT, modes INTEGER) WITH (kafka_topic='ks_hgame_projects_n11_stream', KEY='userid', VALUE_FORMAT='JSON');

Message       
---------------
Table created 
---------------
ksql> show tables;

Table Name                   | Kafka Topic                  | Format | Windowed 
---------------------------------------------------------------------------------
ks_hgame_projects_n111_table | ks_hgame_projects_n11_stream | JSON   | false    
---------------------------------------------------------------------------------
ksql> select * from `ks_hgame_projects_n111_table`;
Invalid Expression ks_hgame_projects_n111_table.ROWTIME.
ksql> select p.userid from `ks_hgame_projects_n111_table` p;
1008
6219
6219
^CQuery terminated
ksql> select p.ROWTIME from `ks_hgame_projects_n111_table` p;
1542766804803
^CQuery terminated
ksql> select * from `ks_hgame_projects_n111_table`;
Invalid Expression ks_hgame_projects_n111_table.ROWTIME.
ksql>
```

**Comment by miguno** [comment]
@tkaven : Which version of KSQL are you using?

Also: Can you try the following variant, where the quotes/backticks around the table name are omitted?

```sql
-- Try this:
select * from ks_hgame_projects_n111_table;

--- instead of
select * from `ks_hgame_projects_n111_table`;
```

**Comment by tkaven** [comment]
> @tkaven : Which version of KSQL are you using?
> 
> Also: Can you try the following variant, where the quotes/backticks around the table name are omitted?
> 
> ```sql
> -- Try this:
> select * from ks_hgame_projects_n111_table;
> 
> --- instead of
> select * from `ks_hgame_projects_n111_table`;
> ```

i find out beause

i use '`' to create:

CREATE TABLE `ks_hgame_projects_n111_table`

...

**Comment by miguno** [comment]
@tkaven : I am bit confused now. :-)

Does the variant below work for you or not?

```sql
-- Try this:
select * from ks_hgame_projects_n111_table;
```

**Comment by tkaven** [comment]
> @tkaven : I am bit confused now. :-)
> 
> Does the variant below work for you or not?
> 
> ```sql
> -- Try this:
> select * from ks_hgame_projects_n111_table;
> ```

hi, select * from ks_hgame_projects_n111_table; not work for me
but if i create use 'CREATE TABLE ks_hgame_projects_n111_table' 
it works!

so in create statement do not use ` every thing is ok

**Comment by miguno** [comment]
@big-andy-coates: Is this something new we need to track on our work on quoted identifiers?

@JimGalasyn: We should consider documenting this.

**Comment by JimGalasyn** [comment]
Tracking with internal ticket KSQL-1938.

**Comment by agavra** [comment]
This should be fixed as of master!
